website/docs/usage/linguistic-features.md

```diff
@@ -578,6 +578,54 @@ import DisplacyEntHtml from 'images/displacy-ent.html'
 
 <Iframe title="displaCy visualizer for entities" html={DisplacyEntHtml} height={275} />
 
+## Entity Linking {#entity-linking}
+
+To ground the named entities into the "real-world",
+spaCy provides functionality to perform entity linking, which resolves a textual entity 
+to a unique identifier from a knowledge base (KB).
+ 
+The default model assigns WikiData identifiers, but you can create your own 
+[`KnowledgeBase`](/api/kb) and [train a new Entity Linking model](/usage/training#entity-linker) using 
+that custom-made KB.
+
+
+### Accessing entity identifiers {#accessing}
+
+The annotated KB identifier is accessible as either a hash value 
+or as a string, using the attributes
+`ent.kb_id` and `ent.kb_id_` of a [`Span`](/api/span) object,
+or the `ent_kb_id` and `ent_kb_id_` attributes of a [`Token`](/api/token) object.
+
+
+```python
+### {executable="true"}
+import spacy
+
+nlp = spacy.load("en_core_web_sm")
```

Comment:
Just adding a quick note here because we still need to decide how we want to be integrating the entity linker with the models.

Reply:
I changed it to "my_custom_el_model" to avoid confusion